docs/public/index.css

```
@@ -154,24 +171,59 @@ pre > code:not([class*='language']) {
   color: inherit;
 }
 
+table,
 pre {
   position: relative;
-  background-color: var(--theme-code-bg);
-  color: var(--theme-code-text);
   --padding-block: 1rem;
   --padding-inline: 2rem;
   padding: var(--padding-block) var(--padding-inline);
   padding-right: calc(var(--padding-inline) * 2);
+  margin-left: calc(var(--padding-inline) * -1);
+  margin-right: calc(var(--padding-inline) * -1);
+  font-family: var(--font-mono);
 
-  line-height: 1.414;
+  line-height: 1.5;
+  font-size: 0.85rem;
   overflow-y: hidden;
   overflow-x: auto;
 }
 
+table {
+  width: 100%; 
+  padding: var(--padding-block) 0;
+  margin: 0;
+  border-collapse: collapse;
+}
+
+/* Zebra striping */
+tr:nth-of-type(odd) { 
+  background: var(--theme-bg-hover);
```

```suggestion
  background: var(--theme-hover-bg);
```

Could this variable be renamed to follow the way the others have been named better?